src/components/calcite-dropdown/calcite-dropdown.tsx

```diff
@@ -40,6 +41,9 @@ export class CalciteDropdown {
     | "center"
     | "end" = "start";
 
+  /** specify the max items to display before showing the scroller, must be greater than 0 **/
+  @Prop() maxItemsToDisplay: number = 0;
```

Comment:
Can't really think of a better name, `displayed-items` ? `items-displayed`?

Reply:
More suggestions: `max-items`, `max-displayed`?

Reply:
Went w/ `max-items`.